test/specs/parsing/functions/functions.tests.ts

```diff
@@ -309,6 +319,19 @@ describe('functions', () => {
 			]);
 		});
 
+		it('returns nothing if the second parameter is not a node', () => {
```

Comment:
Should be 'returns nothing if the context is not a node'

Reply:
https://www.w3.org/TR/xpath-functions-31/#func-id
https://www.w3.org/TR/xpath-functions-31/#func-idref
`If the context item is absentDM31, dynamic error [err:XPDY0002]XP31.`
`If the context item is not a node, type error [err:XPTY0004]XP31.`
In this case, are expectations not compatible with the specifications of the functions(id and id-ref)?